apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -759,7 +759,7 @@ class TopInstructions extends Component {
                 }
               />
             )}
-            {!fetchingData && (
+            {!fetchingData && displayFeedbackTab && (
```

Comment:
This is a strange one, but if you look at what happens when there is feedback but `displayFeedbackTab` is false, it will render the TeacherFeedbackTab anyway even though there's no way to navigate to it or even see it. The invisible textarea in this component _always_ attaches an `onunload` event to the window that warns if you navigate away. It made the page unusable in the case where a student's work was marked 'keep working'